conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,192 @@
+  * Work with Abi to figure out who we work with in CCS and tag in JIRA story.
+    * Consider prioritization (track in JIRA); use template.
+    * Maybe need a separate process for external teams; have UXD Content & CCS just do a quick run-through.
+  * At this point, the UXD Designer is putting designs into Sketch and Marvel and getting feedback on interactions (not content yet). These will not be hi-fi designs.
+  * **Team responsible:** UXD Design (including Dev Tools UXD); Product Management; UXD Content Strategy; CCS
+  * **Time frame:** Conceptual design story (Sprint 1)
+  * **Definition of done:**
+    * UXD Designer creates interaction designs (not hi-fi) in Sketch and Marvel.
+    * Product Manager reviews the interactions and provides feedback.
+    * Stakeholder review (by Product Management and Dev Lead) is complete and all feedback is applied.
+
+#### Phase 2: First draft
+UXD Designer writes the first draft of the quick start, using the following as references:
+* Interaction design plans (to align with user flow)
+* Technical documentation (to avoid contradiction and duplication of effort)
+* Red Hat brand standards/PatternFly UX writing style guide (to align with brand voice and UX writing best practices)
+* CCS style resources (to align with docs best practices)
+* Quick start writing template (to maintain consistency)
+* **Team responsible:** UXD Design
+* **Time frame:** After conceptual design story (Sprint 2)
+* **Definition of done:**
+  * All steps, descriptions, messaging, and other content is completely written.
+  * Draft is ready for CCS review.
+
+#### Phase 3: CCS review
+CCS Technical Writer conducts a review of the draft.
+
+During this step, have a checkpoint/feedback loop with CCS to give UX feedback on existing documentation. A smaller group can work together and identify what it will take to make existing documentation consumable for quick starts.
+
+* **Team responsible:** CCS Technical Writing; UXD Design
+* **Time frame:** Sprint 2
+* **Definition of done:**
+  * CCS Technical Writer reviewed all content and left edits/comments.
+  * UXD Designer addressed all feedback.
+
+#### Phase 4: UXD Content review
+Abi conducts a content review of the draft.
+
+* **Team responsible:** UXD Content Strategy; UXD Design
+* **Time frame:** Sprint 3
+* **Definition of done:**
+  * Abi reviewed all content and left edits/comments.
+  * UXD Designer addressed all feedback.
+  * Stakeholder review (by Product Management and Dev Lead) is complete and all feedback is applied.
+
+#### Phase 5: Implementation
+UXD Developer implements the quick start design and written content in OpenShift.
+
+* **Team responsible:** UXD Design; UXD Development
+* **Time frame:** Initial Development Sprint (Sprint 4)
+* **Definition of done:**
+  * Implementation addresses the main goals of the design and quick start draft.
+  * PR is reviewed and approved to be merged.
```

Comment:
It would be nice to get UX & PM involved during the PR review process. If we test the PR then we can get issues addressed before merging, thus minimize the amount of bugs created.